src/ModalHeader.js

```diff
@@ -33,6 +33,10 @@ ModalHeader.__isModalHeader = true;
 
 ModalHeader.propTypes = {
   /**
+   * The aria-label attribute is used to define a string that labels the current element.
```

Comment:
Perhaps worth noting here that the aria label is for Assistive Technology and is not visibly seen on screen. Otherwise this will be confusing for younger developers that have not dealt with screen readers.